### DISCUSSION TITLE
Pycord url buttons

### DISCUSSION THREAD

**Discussion by BatteTarte** [Pycord-Development/pycord#1933] [Q&A] [answered]
## Hello
I tried to make a redirect url button in my Pycord button view... but it seem not work

    class button(discord.ui.View): 
    @discord.ui.button(label="Url", style=discord.ButtonStyle.url, url = "https://www.google.com/?hl=fr")
    async def button_callback(self, button, interaction):
        await interaction.response.send_message("You clicked the button!")`
But it give me this error `TypeError: button() got an unexpected keyword argument 'url'`

How can i use it?

**Comment by JustaSqu1d** [comment, marked as answer]
You can create a link button by creating a `Button` object and adding it to the view as shown in [this example](https://github.com/Pycord-Development/pycord/blob/master/examples/views/link.py). 

For further reference, please use the [support server](https://pycord.dev/discord) for coding help.

**Reply by BatteTarte** [reply]
Thank you!
